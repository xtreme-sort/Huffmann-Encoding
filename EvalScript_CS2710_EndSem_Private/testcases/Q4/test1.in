GenerateLengthLimitedHuffman
eqyftenmnniysgdixiljfgjzdwdpbkdgrylfactelxznoytqsqgczmigvrwyvtoukvyuoyrpofpmpseyhuhnhnzhbnxtljxyctimiedodxskhobtglukvdzuzgimxgkqtohhjwtlexzqqaagqhgqiroesabgpvowmcuckzcvdxkmxozbbibdhpyfexghsiwoanfoqoofecufcbktppprjddsgbcpxwdootjyiuiqlmzkiokurkbwcojmbjdpodbjzumvvnmjgiheeysqtjojstfkwgqnxugpbfqkznucuimazjljnupnnigvykttpqmashpxffpizbqzadidffdetbdexrsznkkrxhhlopuxbewotckzxzxkrdukxputudcmjcmcgypxupaglyefzdyxgscnltutjipzlhjhbdvoktoslixxlggmzujezietpsnsrfecdlsaqrcddmqhgqdymjhbscsjulual
10
Quit
