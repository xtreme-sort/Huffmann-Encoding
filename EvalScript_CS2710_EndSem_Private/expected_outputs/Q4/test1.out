d 0000
g 0001
o 0010
t 0011
u 0100
x 0101
z 0110
a 01110
b 01111
c 10000
e 10001
f 10010
h 10011
i 10100
j 10101
k 10110
l 10111
m 11000
n 11001
p 11010
q 11011
r 11100
s 11101
y 11110
v 111110
w 111111
cost: 2261
